src/app/components/Button.svelte

```diff
@@ -1,12 +1,11 @@
 <script lang="ts">
-  import { beforeUpdate } from "svelte";
-
   // props
   export let color: string = "grey"; // optional
   export let isOutlined: boolean = false; // optional
   export let hasDropShadow: boolean = false; // optional
   export let size: string = "medium"; // optional
   export let subtext: string = ""; // optional
+  export let data_cy: string = ""; // optional
```

Comment:
Please use `camelCase` when naming new variables in JavaScript.

```suggestion
  export let dataCy: string = ""; // optional
```